services/dask-sidecar/requirements/_base.in

```diff
@@ -15,6 +15,7 @@
 --requirement ../../../packages/models-library/requirements/_base.in
 --requirement ../../../packages/service-library/requirements/_base.in
 --requirement ../../../packages/settings-library/requirements/_base.in
+--requirement ../../../packages/service-integration/requirements/_base.in
```

Comment:
uff I would like to avoid this dependency if possible! Let me catch up with you offline.

Reply:
I actually wonder why because I see no imports from service_integration. did you forget to version something?